marketplace_builder/views/pages/tutorials/partner-portal/assigning-billing-plan-instance.liquid

```diff
@@ -0,0 +1,37 @@
+---
+converter: markdown
+metadata:
+  title: Assigning a Billing Plan to an Instance 
+  description: This guide will help you assign Billing Plans to your clients' Instances. 
+slug: tutorials/partner-portal/assigning-billing-plan-instance
+searchable: true
+---
+
+This guide will help you assign Billing Plans to your clients' Instances. In PlatformOS terms, your clients are your SubPartners, so you'll assign Billing Plans to your SubPartner's Instances. 
+
+## Requirements
+To follow the steps in this tutorial, you have to have at least one Billing Plan created. 
+
+* [Creating a Billing Plan](/tutorials/partner-portal/creating-billing-plan)
+
+## Steps
+
+Assigning a Billing Plan to an Instance is a two-step process:
+
+1. Locate SubPartner Instances 
+2. Assign Billing Plan
+
+### Step 1: Locate SubPartner Instances
+
+In the menu on the left, click on `Partners`, and select your own Partner. The information page of the Partner is displayed. 
+Here you have some information about Instances — your own and your SubPartners'. 
+
+Under "SubPartners Instances", click on the link to the Instances. A table of sold Instances is displayed — these are Instances that your SubPartner has created. Check the row for each Instance: pay attention to the environment (staging or production), and the Billing Plan. 
```

Comment:
Remove "sold"

Reply:
> Check the row for each Instance: pay attention to the environment (staging or production), and the Billing Plan

They won't be able to assign production plan to staging instance and vice versa, so they don't need to pay attention to that. They need to pay attention to remove all "Unbilled" ones.
Maybe I'll be able to add a filter for Plans in the table, but that's not for sure.

Reply:
Thank you! Edited both.